### DISCUSSION TITLE
network support

### DISCUSSION THREAD

**Discussion by alkamida** [compnerd/swift-win32#700] [General]
Hi,
is there any way that we support the Network framework (https://developer.apple.com/documentation/network)?  

Regards

**Comment by compnerd** [comment]
Hmm, that wouldn't be appropriate for this library.  Implementing such an API is certainly possible with Swift, you should hand access to WinSock.  I would say that this is more in the domain of swift-nio but I've had trouble getting the changes needed merges upstream.
